src/test/java/edu/kis/powp/factory/Square.java

```diff
@@ -0,0 +1,29 @@
+package edu.kis.powp.factory;
+
+import java.util.ArrayList;
+
+import edu.kis.powp.command.ComplexCommand;
+import edu.kis.powp.command.DriverCommand;
+import edu.kis.powp.command.OperateToCommand;
+import edu.kis.powp.command.SetPositionCommand;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class Square {
+	Job2dDriver driver;
+
+	public Square(Job2dDriver driver) {
+		this.driver = driver;
+	}
+
+	public void figureScript(){
+		ArrayList<DriverCommand> commands = new ArrayList<>();
+		commands.add(new SetPositionCommand(driver, -120, -120));
+		commands.add(new OperateToCommand(driver, 120, -120));
+		commands.add(new OperateToCommand(driver, 120, 120));
+		commands.add(new OperateToCommand(driver, -120, 120));
+		commands.add(new OperateToCommand(driver, -120, -120));
+
+		ComplexCommand complexCommand = new ComplexCommand(commands);
+		complexCommand.execute();
```

Comment:
This is not a factory. A factory should return a driver command (not execute it). Missing point 3.1.5.